# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../gatos-km/no-uname.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- gatos-km-2004-01-07/Makefile.orig	2005-02-16 01:25:06.789449548 +0100
+++ gatos-km-2004-01-07/Makefile	2005-02-16 01:25:54.731259020 +0100
@@ -56,9 +56,9 @@
 	$(CC) $(CFLAGS) test2.c -o test2
 
 install:
-	mkdir -p /lib/modules/`uname -r`/misc/
-	install km_api_drv.o /lib/modules/`uname -r`/misc/
-	install km_drv.o /lib/modules/`uname -r`/misc/
+	mkdir -p $(MODDIR)/misc/
+	install km_api_drv.o $(MODDIR)/misc/
+	install km_drv.o $(MODDIR)/misc/
 
 tarball:
 	(cd .. ; tar cvf - km/*.[c,h] km/Makefile km/README | gzip - ) > km.tgz
